docs/docsite/rst/scenario_guides/guide_online.rst

```diff
@@ -0,0 +1,29 @@
+*************************
+Using Ansible with Online
+*************************
+
+Introduction
+============
+
+Online is a French hosting company mainly known for providing baremetal servers named Dedibox.
+Check it out: https://www.online.net/en
+
+Dynamic inventory for Online resources
+--------------------------------------
+
+Ansible has a dynamic inventory plugin that can list your resources.
+
+1. Create a YAML configuration such as ``online_inventory.yml``
+
+.. code-block:: yaml
+
+    plugin: online
+
+2. Set your ``ONLINE_TOKEN`` environment variable with your token.
```

Comment:
It might be worth mentioning that you need to open an account and log into it before you can get a token.